hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -1,1060 +1,1178 @@
+be disconnected inside the AST.</td>
+</tr>
+<tr class="even">
+<td>ast2pad_t0_ao, ast2pad_t1_ao</td>
+<td>I/O</td>
+<td></td>
+<td>async</td>
+<td>Analog debug signals. These signals should be connected directly to
+chip PADs. They can share PADs with functional signals but when they are
+used for their analog debug function, the functional I/O must be in
+tri-state.</td>
+</tr>
+<tr class="odd">
+<td><p>dpram_rmf_o,</p>
+<p>dpram_rml_o,</p>
+<p>spram_rm_o,</p>
+<p>sprgf_rm_o,</p>
+<p>sprom_rm_o</p></td>
+<td>O</td>
+<td>dpm_rm</td>
+<td>async</td>
+<td>RAM/ROM Read-write Margin Trimming</td>
+</tr>
+<tr class="even">
+<td>padmux2ast_i</td>
+<td>I</td>
+<td>6</td>
+<td>async</td>
+<td>Digital debug input signals (routed to pin mux). These signals are
+controllable only when DFT features are enabled (Test and RMA states).
+Otherwise, these signals are grounded to 0.</td>
+</tr>
+<tr class="odd">
+<td>ast2padmux_o</td>
+<td>O</td>
+<td>9</td>
+<td>async</td>
+<td>Digital debug output signals (routed to pin mux). These signals are
+only outputted when DFT features are enabled (Test and RMA states).
+Otherwise, these signals are grounded to 0.</td>
+</tr>
+<tr class="even">
+<td>usb_io_pu_cal_o</td>
+<td>O</td>
+<td>20</td>
+<td>async</td>
+<td>USB I/O calibration and trimming</td>
+</tr>
+<tr class="odd">
+<td>io_clk_byp_req_i</td>
+<td>I</td>
+<td>mubi4</td>
+<td>async</td>
+<td><p>External clock mux override request for OTP bootstrap purposes.
+AST responds to the request by setting io_clk_byp_ack_o to 'On'. When
+this bit is set and ack was received, clk_ast_ext_i serves as the io_clk
+clock root.</p>
+<p>Note: When 'On' (after ack), clk_src_io_o clock max frequency is
+limited to 50 MHz</p></td>
+</tr>
+<tr class="even">
+<td>io_clk_byp_ack_o</td>
+<td>O</td>
+<td>mubi4</td>
+<td>async</td>
+<td>AST response to io_clk_byp_req_i. The ack is set to 'On' after clock
+switching function is performed.</td>
+</tr>
+<tr class="odd">
+<td>all_clk_byp_req_i</td>
+<td>I</td>
+<td>mubi4</td>
+<td>async</td>
+<td><p>External clock mux override request for OTP bootstrap purposes.
+AST responds to the request by setting io_clk_byp_ack_o to 'On'. When
+this bit is set and ack was received, clk_ast_ext_i serves as the io_clk
+clock root.</p>
+<p>Note: When 'On' (after ack), clk_src_io_o clock max frequency is
+limited to 50 MHz</p></td>
+</tr>
+<tr class="even">
+<td>all_clk_byp_ack_o</td>
+<td>O</td>
+<td>mubi4</td>
+<td>async</td>
+<td>AST response to io_clk_byp_req_i. The ack is set to 'On' after clock
+switching function is performed.</td>
+</tr>
+<tr class="odd">
+<td>ext_freq_is_96m_i</td>
+<td>I</td>
+<td>mubi4</td>
+<td>async</td>
+<td>External clock frequency indication to AST. When set, it indicates
+that the external clock is 96MHz.</td>
+</tr>
+<tr class="even">
+<td>lc_dft_en_i</td>
+<td>I</td>
+<td>lc_tx</td>
+<td>async</td>
+<td><p>DFT enable.</p>
+<p>Signal type is lc_tx_t (see lc_ctl_pkg.sv)</p>
+<p>On = 4'b1010; Off = 4'b0101</p></td>
```

Comment:
note: this does not seem to be correct, see: https://github.com/lowRISC/opentitan/blob/71f19edb3fd7331291abd6c2bbe5c51557faba39/hw/ip/lc_ctrl/rtl/lc_ctrl_pkg.sv#L41-L44